website/docs/documentation/Pipelines/Illumina_Genotyping_Arrays_Pipeline/Illumina_array_spec.md

```diff
@@ -0,0 +1,111 @@
+# VCF Overview: Illumina Genotyping Array 
+
+The [Illumina Genotyping Array Pipeline](https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/genotyping/illumina/IlluminaGenotypingArray.wdl) v1.11.0 pipeline produces a VCF (Variant Call Format) output with data processing and sample-specific genotype information. The VCF follows the format listed in the VCF 4.2 specification, but additionally contains fields and attributes that are unique to the Arrays pipeline.
+
+This document describes the Array pipeline’s unique VCF fields and attributes that are not listed in the standard VCF specification. To learn more about the pipeline, see the [Illumina Genotyping Array Pipeline Overview](./IlluminaGenotypingArray.documentation.md).
+
+
+:::tip How do I view a VCF file?
+A VCF is a tab delimited text file that you can open with either a text editor or with a spreadsheet editor (like Microsoft Excel). 
+:::
+
+## Meta information fields
+Each VCF has meta information fields with attributes that generally describe the processing of the sample in the array. 
+
+
+- arrayType - Name of the genotyping array that was processed.
+- autocallDate - Date that the genotyping array was processed by ‘autocall’ (aka gencall), the Illumina genotype calling software
+- autocallGender - Gender (sex) that autocall determined for the sample processed
+- autocallVersion - Version of the autocall/gencall software used
+- chipWellBarcode - Chip well barcode (a unique identifier for sample as processed on a specific location on the Illumina genotyping array)
+- clusterFile - Cluster file used
+- extendedIlluminaManifestVersion - Version of the ‘extended Illumina manifest’ used by the VCF - generation software.
+- extendedManifestFile - File name of the ‘extended Illumina manifest’ used by the VCF generation software
+- fingerprintGender - Gender (sex) determined using an orthogonal fingerprinting technology, populated by an optional parameter used by the VCF generation software
+- gtcCallRate - GTC call rate of the sample processed that is generated by the autocall/gencall software and represents the fraction of callable loci that had valid calls
+- imagingDate - Creation date for the chip well barcode IDATs (raw image scans)
+- manifestFile - Name of the Illumina manifest (.bpm) file used by the VCF generation software
+- sampleAlias - Sample name
+
+:::tip Illumina-specific meta information
+In addition to the above attributes, there are several attributes specific to Illumina control values that are not described here: Biotin, DNP, Extension, Hyb, NP, NSB, Restore, String, TargetRemoval. 
+:::
+
+## Header line columns 
+Following the meta-information, there are 8 standard header line columns:
+1. #CHROM 
+2. POS 
+3. ID 
+4. REF 
+5. ALT 
+6. QUAL 
+7. FILTER 
+8. INFO
+
+These columns are standard to each VCF, but the Illumina Genotyping Array Pipeline has fields with additional attributes as described below:
```

Comment:
Actually, the attributes below are not separate fields, they are values that can be found in the particular column.  For instance the Filter column may contain the fields/attributes listed in the next section.

Reply:
@gbggrant Thanks for clarifying. Will update this and also add that pipeline VCS has an additional format column.